### PR TITLE
Add support for default (wildcard) error page

### PR DESCRIPTION
**Pull request by evvvvr**
Add ability to specify default (fallback) error page in Caddyfile  via '*' as status code. This addresses #752

### CODE REVIEW

File: caddyhttp/errors/setup.go [hunk 1]
```diff
@@ -122,11 +122,15 @@ func errorsParse(c *caddy.Controller) (*ErrorHandler, error) {
...
-				if err != nil {
-					return hadBlock, c.Err("Expecting a numeric status code, got '" + what + "'")
+				if what == "*" {
+					handler.GenericErrorPage = where
+				} else {
+					whatInt, err := strconv.Atoi(what)
+					if err != nil {
+						return hadBlock, c.Err("Expecting a numeric status code or '*', got '" + what + "'")
+					}
+					handler.ErrorPages[whatInt] = where
```
Comment (vchimishuk) [inline caddyhttp/errors/setup.go]: Maybe it is better to add checking for status code duplications? It can prevent some stupid copy-paste user errors.

Comment (evvvvr) [inline caddyhttp/errors/setup.go]: @vchimishuk Probably, but it should be a matter of different issue

Comment (mholt) [inline caddyhttp/errors/setup.go]: I agree, let's not forget about that now that you've brought it up, but it doesn't need to go in this PR per-se.

**Comment by mholt** [comment]
This is looking great! Thank you so much @evvvvr.

**Comment by mholt** [comment]
@evvvvr I'll give you collaborator status so you can continue to help with Caddy as you are able to - it'll be easier for you now since you can create branches on this repository instead of having to fork the project. Also, please remember to review the [contributing guidelines](https://github.com/mholt/caddy/blob/master/CONTRIBUTING.md) - most notably, that we like to have at least 2 collaborators approve a PR before it is merged, and to squash PRs with multiple small commits.

Really appreciate your contribution, again!

**Comment by evvvvr** [comment]
@mholt Thank You!

**Pull request merged**